data/mods/Russ_Extra_Stuff/items/container-Buckets.json

```diff
@@ -0,0 +1,82 @@
+[
+ {
+        "id": "bucket_storage",
+        "type": "CONTAINER",
+        "symbol": ")",
+        "color": "dark_gray",
+        "name": "5 gallon bucket",
+        "category": "other",
+        "description": "A five gallon bucket with a lid like the kind you used to fill with mackerel when you were fishing with your father.  Could be used for liquids or storage, just activate it to switch between modes.  Right now you're using it for wearable storage.",
+        "price": 1000,
+        "weight": 155,
+        "volume": 7,
+        "bashing": 1,
+        "cutting": 0,
+        "to_hit": -1,
+        "material": "plastic",
+        "contains": 75,
+        "armor_data" : {
+        "covers" : ["HANDS"],
+        "coverage" : 5,
+        "material_thickness" : 4,
+        "encumbrance" : 30,
+        "environmental_protection" : 0,
+        "warmth" : 0,
+        "storage" : 25
+        			  },
+        "rigid": true,
+        "watertight": true,
+        "seals": true,
+        "flags": ["STURDY"],
+
+    "use_action": {
+        "type": "auto_transform",
```

Comment:
This should probably be just "transform" without the "auto_". Otherwise it may act weird in some cases.